packages/sirius-web/backend/sirius-web-domain/src/main/java/org/eclipse/sirius/web/domain/boundedcontexts/library/Library.java

```diff
@@ -57,7 +55,7 @@ public class Library extends AbstractValidatingAggregateRoot<Library> implements
     private AggregateReference<SemanticData, UUID> semanticData;
 
     @MappedCollection(idColumn = "library_id", keyColumn = "index")
-    private Set<LibraryDependency> dependencies = new LinkedHashSet<>();
+    private List<LibraryDependency> dependencies = new ArrayList<>();
```

Comment:
This needs to be a list to support index.
The issue wasn't visible in the previous PR because we don't test libraries with dependencies.

Reply:
:+1: